File: Coding/Ruby/sum_3_and_5.rb
# Program to get a number n from the user and calculate the sum from 1 to n

puts "Hello user please enter a number to sum to: "
n = gets.to_i
sum = 0

for i in 1 .. n
    if ((i%3 == 0) or (i%5 == 0))
        sum += i
    end
end

puts "The sum of the numbers 1 to #{n} is #{sum}."

File: Coding/C++/sum_3_and_5.cpp
/* Program that asks the user for a number and calculates the sum of numbers from 1 to the entered number */

#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int n,sum,i;

    sum = 0;

    cout<<"Hello user please enter the number n to sum up to: ";
    cin>>n;

    for (i = 0; i <= n; i++){
        if ((i%3 == 0) && (i%5 == 0)){
            sum = sum + i;
        }
    }

    cout<<"The sum of the numbers 1 to " << n << " is: " << sum << " \n";

    return 0;
}